### DISCUSSION TITLE
Security Onion 2.3.* Full Elastic Stack docker images

### DISCUSSION THREAD

**Discussion by z3r0RuSh** [Security-Onion-Solutions/securityonion#3745] [General] [answered]
I am currrntly working in an environment that has recieved full elastic licensing and was wondering if there will be a release of docker images like 16.04 had where we have full control of the stack in Kibana to support all the addtional features.

**Comment by defensivedepth** [comment, marked as answer]
Elastic recently changed their licensing (https://blog.securityonion.net/2021/02/elastic-license-changes-and-security.html).

With 2.3.40, the changes described in that post went into effect, and you are now asked to accept the Elastic license during setup. Setup now installs the Features version of the Elastic Stack - which is what I think you are looking for.
